assets/src/components/d3/createProgressBar.js

```diff
@@ -104,6 +110,8 @@ function createProgressBar ({ data, width, height, el, tip }) {
 
   if (tip) {
     tip.direction('s')
+      .offset([0, 60])
+      .style('max-width', '300px')
```

Comment:
I added an offset and a max-width of 300px to the tooltip to prevent the overflow issue for now